Client/Frontend/Login Management/BreachAlertsClient.swift

```diff
@@ -37,10 +60,21 @@ public class BreachAlertsClient: BreachAlertsClientProtocol {
                 completion(Maybe(failure: BreachAlertsError(description: error.localizedDescription)))
                 return
             }
-            guard let data = data, !data.isEmpty else {
-                completion(Maybe(failure: BreachAlertsError(description: "empty data")))
+            guard let data = data else {
+                completion(Maybe(failure: BreachAlertsError(description: "invalid data")))
                 return
             }
+
+            let httpResponse = response as? HTTPURLResponse
+            guard let etag = httpResponse?.allHeaderFields["Etag"] as Any as? String else { return }
```

Comment:
Log to sentry and assert(false) to abort in debug builds so we alert us something is very broken.